src/Main/NavBar.jsx

```diff
@@ -4,6 +4,28 @@ import logo from "../assets/images/assembo-icon-small.png";
 import Box from '@mui/material/Box';
 import Toolbar from '@mui/material/Toolbar';
 import IconButton from '@mui/material/IconButton';
+import Button from '@mui/material/Button';
+import axios from '../axios';
+
+const google=window.google;
```

Comment:
can we have some comments on this?
- explain how we are able to access window.google
- explain this is a 3rd party code, which is why we are exposing a object via the window object